index.html

```diff
@@ -3433,63 +3433,177 @@ <h1 id="title" class="text-xl font-bold">Omnispace</h1>
+                      class="mr-3 min-h-[20px] min-w-[20px] text-gray-400 group-hover:text-gray-500"
+                      fill="none"
+                      stroke="currentColor"
+                      stroke-width="1.5"
+                      viewBox="0 0 24 24"
+                      xmlns="http://www.w3.org/2000/svg"
+                      aria-hidden="true"
+                    >
+                      <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        d="M9 15L3 9m0 0l6-6M3 9h12a6 6 0 010 12h-3"
+                      ></path>
+                    </svg>
+                    Reply
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="mt-4 flex items-end gap-x-2">
+            <div class="w-full">
+              <template x-if="replyingToMessage">
+                <div
+                  x-data="{
+                  get senderName() {
+                    const roomId = replyingToMessage['event']['room_id'];
+                    const senderId = replyingToMessage['event']['sender'];
+                    const members = roomList.get(roomId).getMembers();
+                    const senderName = members.filter((member) => member.userId === senderId)[0].rawDisplayName;
+                    return senderName;
+                  },
+                  get messageContent() {
+                    const isReply = replyingToMessage.event.content?.['m.relates_to']?.['m.in_reply_to'];
+                    const msg = replyingToMessage.event.content.body;
+                    if (isReply) {
+                      // remove matrix part of who you're replying to
+                      // (e.g. '> <@user:matrix.org> ')
+                      return msg.replace(/> <[^>]+>\s?/, '')
+                    }
+                    return msg;
+                  }
+                }"
+                  class="flex items-center divide-x gap-x-2 px-2.5 py-1 border-x border-t rounded-tl-md rounded-tr-md w-full"
+                >
+                  <svg
+                    class="h-5 w-5 min-h-[20px] min-w-[20px] text-gray-400 group-hover:text-gray-500"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-width="1.5"
+                    viewBox="0 0 24 24"
+                    xmlns="http://www.w3.org/2000/svg"
+                    aria-hidden="true"
+                  >
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      d="M9 15L3 9m0 0l6-6M3 9h12a6 6 0 010 12h-3"
+                    ></path>
+                  </svg>
+
+                  <div class="flex flex-col pl-2 w-[calc(100%-30px)]">
+                    <p x-text="senderName" class="font-bold"></p>
+                    <p x-text="messageContent" class="truncate"></p>
+                  </div>
```

Comment:
this part doesnt seem to show up when i try to reply to a reply

Reply:
any errors in the console?

Reply:
did removing the `<pre>` have something to do with it?

Reply:
<pre> was there just for seeing the data of `replyingToMessage` object, shouldn't have affect anything

Reply:
lack of hot module replacement and these cache issues is super annoying